egs/lre07/v2/local/dnn/README

```diff
@@ -0,0 +1,5 @@
+This directory contains DNN scripts based on the nnet2 recipes found in
+the ASR examples (e.g., fisher_english). The scripts have been modified
+for speaker recognition purposes. Most of the scripts are lightly modified
```

Comment:
Can you change the wording, to refer to language-id instead of speaker recognition?